test/index.test.js

```diff
@@ -0,0 +1,103 @@
+describe('Web test', () => {
+  let server
+  let claimRepositoryMock
+
+  beforeEach(async () => {
+    claimRepositoryMock = require('../server/repository/claim-repository')
+    jest.mock('../server/repository/claim-repository')
+    jest.mock('../server/repository/minetype-repository')
+    jest.mock('../server/services/message-service')
+  })
+
+  test('GET / route returns 404', async () => {
+    const options = {
+      method: 'GET',
+      url: '/'
+    }
+    let createServer = require('../server')

```

Comment:
Move the `require` to the top of the file. There's no need to call this again in each test.